tests/manifold/spherical_manifold_04.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::Volume:       14109.1
```

Comment:
It is also a test that takes unnecessarily long to run... ;) I know it's better with 4, but if the bug was already caught with 1, why bother in making it run longer?

With `1`:
~~~
Test project /usr/local/src/d2-parallel-package/dealii/build
    Start 502: manifold/spherical_manifold_04.debug
1/2 Test #502: manifold/spherical_manifold_04.debug .....   Passed    4.95 sec
    Start 503: manifold/spherical_manifold_04.release
2/2 Test #503: manifold/spherical_manifold_04.release ...   Passed    4.93 sec
~~~
with `4`:
~~~
Test project /usr/local/src/d2-parallel-package/dealii/build
    Start 502: manifold/spherical_manifold_04.debug
1/2 Test #502: manifold/spherical_manifold_04.debug .....***Failed   15.19 sec
    Start 503: manifold/spherical_manifold_04.release
2/2 Test #503: manifold/spherical_manifold_04.release ...***Failed    11.09 sec
~~~